bot/common.py

```diff
@@ -114,13 +114,15 @@ def setup_db():
     global readonly_db
     if readonly_db is not None:
         readonly_db.close()
-    readonly_db = pg.connect(settings.readonly_connection_string)
+    readonly_db = pg.connect(settings.readonly_connection_string,
+                             application_name="mb2wikidatabot readonly")
```

Comment:
Since this may end in logs (https://www.postgresql.org/docs/current/static/runtime-config-logging.html#GUC-APPLICATION-NAME) i would keep it shorter, and without any space: `"mb2wikidatabot_RO"` and `"mb2wikidatabot_RW"`

Reply:
I used short names like that at first, but decided against it because the difference is only a single letter.

Reply:
> I used short names like that at first, but decided against it because the difference is only a single letter.

It was more about the whitespace than the length.